src/plots/areaPlot.ts

```diff
@@ -119,6 +118,10 @@ export class Area<X> extends Line<X> {
   }
 
   protected _additionalPaint() {
+    if (this.renderer() === "canvas") {
```

Comment:
the area plot refers to an additional SVG drawstep for the line. this throws exceptions since when we don't use SVG. so, i just shortcircuit in these cases.